src/pages/services/aem-universal-editor/api/modal/index.md

```diff
@@ -44,7 +44,8 @@ function ExtensionRegistration() {
                       icon: 'PublishSchedule',
                       onClick: async () => {
                         console.log('Button has been pressed.');
-                        url: "/index.html#/modal", // absolute or relative path
+                        // absolute or relative path
+                        const url: "/index.html#/modal"; 
```

Comment:
I think it should be `const url= "/index.html#/modal"; // absolute or relative path`